locales/lt/json.json

```diff
@@ -243,7 +243,7 @@
     "El Salvador": "Salvadoras",
     "Email": "El. pašto adresas",
     "email": "el. pašto adresas",
-    "Email Address": "el",
+    "Email Address": "El. paštas",
```

Comment:
Should the "email" and "email address" fields be translated to the same form? What do you think?

Reply:
Resolved in https://github.com/Laravel-Lang/lang/pull/4915